includes/class-freemius.php

```diff
@@ -1380,16 +1380,35 @@ function _run_garbage_collector() {
         }
 
         /**
-         * Opens the support forum subemenu item in a new browser page.
+         * Modifies all external links in the submenu by altering their href, and also opens them in new tab if needed.
          *
          * @author Vova Feldman (@svovaf)
+         * @author Swashata Ghosh (@swashata)
          * @since  2.1.4
          */
-        static function _open_support_forum_in_new_page() {
+        static function _handle_submenu_external_link() {

```

Comment:
This now handles external links and opening new tab for any SDK introduced submenu link. @fajardoleo please also help test with tabs.